<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>23</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Compose</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inbox</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sent</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Draft</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Trash</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Account</string>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>481</width>
     <height>451</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="Compose">
    <widget class="QLineEdit" name="lineEditToUserCompose">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSubjectCompose">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSendCompose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSaveCompose">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAttachFilerCompose">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>AttachFilers</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCompose">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditContentCompose">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>144</y>
       <width>451</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="InBox">
    <widget class="QTableWidget" name="InBoxTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>441</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="InBoxInfo">
    <widget class="QLineEdit" name="lineEditSenderInbox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSubjectInBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTimeInBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserInBox">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>111</y>
       <width>381</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDownloadAttachFile">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>AttachFile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReturnInBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboInBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Sender</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Sent">
    <widget class="QTableWidget" name="SentTable">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>361</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SentInfo">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRecSent">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSubSent">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTimeSent">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserSent">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Draft">
    <widget class="QTableWidget" name="DraftTable">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DraftInfo">
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRecDraft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSubDraft">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTimeDraft">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserDraft">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Delete">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="account">
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>unread count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="count">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
